EIPS/eip-4527.md

```diff
@@ -4,7 +4,7 @@ title: QR Code data transmission protocol for the offline wallets
 description: QR Code data transmission protocol between wallets and offline signers.
 author: Aaron Chen (@aaronisme), Sora Lee (@soralit), ligi (@ligi), Dan Miller (@danjm), AndreasGassmann (@andreasgassmann), xardass (@xardass), Lixin Liu (@BitcoinLixin)
 discussions-to: https://ethereum-magicians.org/t/add-qr-code-scanning-between-software-wallet-cold-signer-hardware-wallet/6568
-status: Stagnant
+status: Review
```

Comment:
Must be moved back to draft, since it never made it to review before it was stagnated.

Reply:
OK

Reply:
@Pandapip1 Hi, I have moved it back to draft, please check it out.